### PR TITLE
Deliverable 2

### PR DESCRIPTION
**Pull request by s1nuh3**
All points to deliverable 2 are applied 

Create a client to consume an external API
Add an endpoint to consume the external API client
The information obtained should be stored in the CSV file
Add unit testing* Some more test are in progress
Update the endpoint made in the first deliverable to display the result as a JSON

### CODE REVIEW

File: service/user/user.go [hunk 1]
```diff
@@ -99,22 +71,25 @@ func convertUsers(rcd [][]string) (*[]model.Users, int, error) {
...
-	u := model.Users{
-		ID:     id,
-		Name:   r[1],
-		Email:  r[2],
-		Gender: r[3],
-		Status: r[4],
+
+	for _, r := range rcd {
+		if r[0] == strconv.Itoa(id) {
+			u, _ = parseUserRecord(r)
```
Comment (alvarocueva) [inline service/user/user.go]: Handle the error

**Comment by alvarocueva** [comment]
I see good advance.
Also, I know you have a lot WIP stuff, just don't forget to follow best practices.

File: service/user/user.go (hunk 1 above)
Comment (s1nuh3) [reply service/user/user.go]: Hi! This is a integer to string, that doesn't returns an error.

**Comment by s1nuh3** [comment]
I passed some more changes, adding unit testing for ImportHandler and some more err checks

**Pull request merged**